Traceback (most recent call last):
  File "/data/yjx/code_for_first_task/PSOWithcoda/VulnerabilityPrediction/Comparative_Experiment/test.py", line 219, in <module>
    main()
  File "/data/yjx/code_for_first_task/PSOWithcoda/VulnerabilityPrediction/Comparative_Experiment/test.py", line 79, in main
    config = config_class.from_pretrained(args.model_name_or_path, cache_dir=args.cache_dir if args.cache_dir else None)
AttributeError: 'Namespace' object has no attribute 'model_name_or_path'
